ArduPlane/Log.cpp

```diff
@@ -287,6 +326,36 @@ void Plane::Log_Write_Vehicle_Startup_Messages()
     gps.Write_AP_Logger_Log_Startup_messages();
 }
 
+/*
+  log a COMMAND_INT message
+ */
+void Plane::Log_Write_MavCmdI(const mavlink_command_int_t &mav_cmd)
+{
+    const char *name = "CMDI";
```

Comment:
There is no point in changing the name, the command has already been logged. We don't need to create 3 more log entries for it.

Reply:
This last-minute name-change before logging from CMDI to other similar name/s for the three relevant slew controls of Heading, Speed, and Altitude means that although they all log in the same format, they are easier to graph as separate data points with log analyser/s  than if they were all called 'CMDI'. ( which is the default , and used for all other cmd-long packets ) . we don't log things three times, we just log three variants.